lektor_tinymce.py

```diff
@@ -51,6 +51,11 @@ def patched_endpoint(*args, **kwargs):
     )
 
 
+def url_rule_wrapper(rule, *args, **kwargs):
+    local_rule = rule
```

Comment:
Is this assignment necessary? Seems like `rule` should be captured in the closure anyway, but because of all the black magic you're doing in `on_server_spawn`, I'm not quite sure 😅

Reply:
Without copying `rule` (which comes from `cell_contents`), the value of `rule` will always be the last value of `cell_contents` in loop. It looks like some magic with `__closure__` and `cell_contents`.